spec.md

```diff
@@ -232,12 +233,14 @@ Policies MUST specify the following fields:
   for this domain might be handled by any MX whose hostname is a subdomain of
   "example.com" or "example.net."
 * a: The mechanism to use to authenticate this policy itself. See the section
-  _Policy_ _Authentication_ for more details. Possible values are:
-  * webpki:URI, where URI points to an HTTPS resource at the recipient domain
-    that serves the same policy text.
+  _Policy_ _Discovery_ _&_ _Authentication_ for more details. Possible values are:
+  * webpki: Indicating that the policy is expected to be served over HTTPS on a
```

Comment:
Why this change? I don't think it matters much one way or another, but this is a slightly larger change that should maybe be called out (or made its own pull request). 

We also should not call this a "well-known" URI if we don't plan to use .well-known (RFC 5785), which means we can remove that consideration from the IANA considerations section, right?